website/registrations/migrations/0022_auto_20200325_2045.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 3.0.3 on 2020-03-25 19:45
+
+import django.core.validators
+from django.db import migrations, models
+
+
+def set_contribution_fields(apps, schema_editor):
+    """On this migration, set the empty contribution fields to the paid contribution."""
+    Entry = apps.get_model('registrations', 'Entry')
+    for entry in Entry.objects.all():
+        if entry.payment and not entry.contribution:
+            entry.contribution = entry.payment.amount
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('registrations', '0021_auto_20191106_2338'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='entry',
+            name='contribution',
+            field=models.FloatField(validators=[django.core.validators.MinValueValidator(7.5)], verbose_name='contribution'),
+        ),
+        migrations.RunPython(set_contribution_fields)
```

Comment:
This should be split in a separate migration, running python with a database change has not worked in the past because it is no longer atomic.